float64 lat_x
float64 long_y
float64 alt_z